here-naksha-lib-psql/src/main/java/com/here/naksha/lib/psql/PostgresStorage.java

```diff
@@ -450,6 +439,50 @@ synchronized void initStorage(@NotNull PsqlStorage.Params params, @NotNull IoHel
     }
   }
 
+  private void initStoragePlv8(
+      PsqlStorage.@NotNull Params params, @NotNull IoHelp ioHelp, Statement stmt, PsqlConnection conn)
+      throws SQLException {
+    final Plv8Env plv8Env = new Plv8Env();
+    plv8Env.install(conn, latest.toLong());
+    conn.commit();
+  }
+
+  private void initStoragePlpgsql(
+      PsqlStorage.@NotNull Params params, @NotNull IoHelp ioHelp, Statement stmt, PsqlConnection conn)
```

Comment:
Since `params` became member property we don't need them as argument (the client-side unmodified passes member instance)

Reply:
Fixed.